src/multifile.jl

```diff
@@ -181,12 +181,33 @@ maskingvalue(mfds::MFDataset) = maskingvalue(mfds.ds[1])
 Base.parent(v::MFVariable) = v.var
 Base.parent(v::MFCFVariable) = v.var
 Base.Array(v::MFVariable) = Array(parent(v))
-Base.getindex(v::MFVariable,indexes::TIndices...) = getindex(parent(v),indexes...)
-Base.setindex!(v::MFVariable,data,indexes::TIndices...) = setindex!(parent(v),data,indexes...)
+
+function DiskArrays.readblock!(v::MFVariable{T, N}, aout,indexes::Vararg{OrdinalRange, N}) where {T, N}
+    aout .= parent(v)[indexes...]
+end
+function DiskArrays.readblock!(v::MFCFVariable{T, N}, aout, indexes::Vararg{OrdinalRange, N}) where {T, N}
+    aout .= v.cfvar[indexes...]
+end
+
+function DiskArrays.writeblock!(v::MFVariable{T,N}, data, indexes::Vararg{OrdinalRange, N}) where {T, N}
```

Comment:
```suggestion
function DiskArrays.writeblock!(v::MFVariable{T,N}, data) where {T, N}
```

This was all I meant, I should have just added the code